docker-compose.yml

```diff
@@ -130,12 +144,15 @@ services:
     security_opt:
       - "no-new-privileges:true"
   web-server:
-    image: "sigopt/web:${TAG}"
+    build:
+      dockerfile: docker/images/web/Dockerfile
+      args:
+        - NODE_VERSION
     environment:
       - "sigopt.web.cookiejar_credentials.secretAccessKey=$MINIO_ROOT_PASSWORD"
       - "sigopt_server_config_file=$sigopt_server_config_file"
       - "NODE_EXTRA_CA_CERTS=/sigopt-server/artifacts/tls/root-ca.crt"
-      - "sigopt_server_version=git:$TAG"
+      - sigopt_server_version
```

Comment:
looks like this is getting set to null? intentional?

Reply:
This just forwards the environment variable from the host